scripts/cubemx-importer.stanza

```diff
@@ -118,28 +127,38 @@ defmethod print (o:OutputStream, s:SwdOption):
   println(os, "swd.swdclk => self.%_" % [pin-name(swclk(s))])
 
 defmethod print (o:OutputStream, j:JtagOption):
-  val jtag = "jtag" when trstn(j) is-not False else "jtag-4"
+  val jtag = "jtag()" when trstn(j) is False else "jtag([JTAG-TRSTN])"
   println(o, "supports %_:" % [jtag])
   val os = IndentedStream(o)
   println(os, "%_.tck => self.%_" % [jtag, pin-name(tck(j))])
   println(os, "%_.tdi => self.%_" % [jtag, pin-name(tdi(j))])
   println(os, "%_.tdo => self.%_" % [jtag, pin-name(tdo(j))])
   println(os, "%_.tms => self.%_" % [jtag, pin-name(tms(j))])
-  if jtag == "jtag":
-    println(os, "jtag.trstn => self.%_" % [pin-name $ trstn(j) as Stm32Pin])
+  if trstn(j) is-not False:
+    println(os, "%_.trstn => self.%_" % [jtag, pin-name $ trstn(j) as Stm32Pin])
 
 defmethod print (o:OutputStream, opt:GpioOption):
   println(o, "supports gpio:")
   val os = IndentedStream(o)
   println(os, "gpio.gpio => self.%_" % [pin-name(gpio(opt))])
 
+defmethod print (o:OutputStream, opt:AdcOption):
+  println(o, "supports adc:")
+  val os = IndentedStream(o)
+  println(os, "adc.adc => self.%_" % [pin-name(adc(opt))])
```

Comment:
`print`, not `println`

Reply:
We need println. Each line is separate

Reply:
last line should be print, not println

Reply:
To add some context,
`print` should always end without a new line since `println` calls `print` then adds a new line.

Reply:
No, we do not want any 'artificial' blank line

Reply:
This is how the output looks and it is how we want it
```
    supports dac:
      dac.dac => self.PA[4]

    supports dac:
      dac.dac => self.PA[5]

```

Reply:
why not printing a blank line where you call the print function that prints 
```
supports dac:
      dac.dac => self.PA[4]
```
?

Reply:
I have also seen a `lnprintln` function. I think I have seen @ariel-hi using it if I am not mistaken.

Reply:
Yes I use `lnprintln` in the KiCad Importer. But it's not natively supported.